src/components/Home.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Home = () => {
+  return (
+    <div>
+      <h2 className="header-text">NC Games Review </h2>
+    </div>
+  );
+};
+
```

Comment:
need a link to reviews on home if youre going to have it on a /reviews path

Reply:
I am thinking later on I may display the top 10 reviews on the home page